codersdk/features.go

```diff
@@ -75,12 +75,14 @@ type Feature struct {
 }
 
 type Entitlements struct {
-	Features     map[FeatureName]Feature `json:"features"`
-	Warnings     []string                `json:"warnings"`
-	Errors       []string                `json:"errors"`
-	HasLicense   bool                    `json:"has_license"`
-	Experimental bool                    `json:"experimental"`
-	Trial        bool                    `json:"trial"`
+	Features   map[FeatureName]Feature `json:"features"`
+	Warnings   []string                `json:"warnings"`
+	Errors     []string                `json:"errors"`
+	HasLicense bool                    `json:"has_license"`
+	Trial      bool                    `json:"trial"`
+
+	// DEPRECATED: use Experiments instead.
+	Experimental bool `json:"experimental"`
```

Comment:
Would be nice to return experiments here as well to avoid the extra frontend request on each page load and the extra API endpoint

Reply:
We can, but I want to be careful about conflating Features (which are, as far as I'm aware, enterprise-only) with Experiments.